src/utils/fastapi.py

```
@@ -12,12 +15,14 @@
 
 
 async def get_logged_in() -> bool:
-    jwt = r.get(LOGIN_KEY)
+    jwt: str = r.get(LOGIN_KEY)  # type: ignore
     if not jwt: return False
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)